events/2022-google-summer-of-code/index.markdown

```diff
@@ -85,9 +85,21 @@ For general MoveIt GSoC questions you can contact [Mark Moll](mailto:mark@pickni
 * Preferred skills: MoveIt, Collision Checking, and C++
 * Mentor: [Simon Schmeisser](mailto:simon.schmeisser@optonic.com), [Michael Görner](mailto:goerner@informatik.uni-hamburg.de)
 
+### Better Parallel Planning with MoveIt 2
+* One of the latest features of MoveIt 2 is the [parallel planning API for moveit_cpp](https://moveit.picknik.ai/main/doc/how_to_guides/parallel_planning/parallel_planning_tutorial.html). It enables using multiple planning pipelines in parallel, choosing a stopping criteria and a solution selection function. While the initial implementation is finished and the feature is fully operational, there is a still room for improvements. Here is s list of ideas in which ways the feature could be developed further over the course of a GSoC project:
+  * Enable parallel planning in the [move_group interface](https://moveit.picknik.ai/main/doc/examples/move_group_interface/move_group_interface_tutorial.html)
+  * Make parallel planning usable in [RVIZ Motion Planning Panel](https://moveit.picknik.ai/main/doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.html)
+  * Visualize different parallel planning solutions in RVIZ
+  * Improve logging of parallel planning processes
+  * Add additional default solution selection functions
+  * Your idea to make parallel planning with MoveIt 2 better
+* This list is neither complete, nor need all items be addressed in your project. In addition to choosing some items for a project proposal, it would be very excited to hear your improvement idea(s).
```

Comment:
```suggestion
* This list is neither complete, nor need all items be addressed in your project. In addition to choosing some items for a project proposal, it would be very exciting to hear your improvement idea(s).
```